apisix/core/config_yaml.lua

```diff
@@ -69,9 +68,36 @@ local mt = {
     end
 }
 
+local apisix_config
+local apisix_config_mtime
+local apisix_config_path
+
+local apisix_yaml_path = profile:yaml_path("apisix")
+local apisix_json_path = ngx.re.sub(apisix_yaml_path, [[\.yaml$]], ".json", "jo")
+local file_configs = {
+    {path = apisix_yaml_path, type = "yaml"},
+    {path = apisix_json_path, type = "json"}
+}
+
+local function get_config_file_info()
+    local paths_str = ""
```

Comment:
The variable name is not clear, it is only used to record errors.

This function has many return values, you can consider using a table.

refer to:
```
local function get_config_file_info()
    local not_found_paths = {}

    for _, config in ipairs(file_configs) do
        local attr, err = lfs.attributes(config.path)
        if attr then
            return config.path, config.type, attr.modification
        else
            table.insert(not_found_paths, config.path)
            log.warn("failed to fetch attributes for ", config.path, ": ", err)
        end
    end

    local paths_str = table.concat(not_found_paths, ", ")
    return nil, nil, nil, "Failed to find any configuration file with path " .. paths_str
end
```

Reply:
After communicating with membphis and others, I found that it needed to be reconstructed, and the current one is indeed not good enough.